src/components/categories/weather/models/BotMessage.vue

```diff
@@ -0,0 +1,32 @@
+<template>
```

Comment:
Moim zdaniem powinien to być komponent z obrazkiem, żeby nie trzeba było używać zawsze obu razem:) Z sourcem obrazka, możemy też użyć sprytnych defaultów: https://stackoverflow.com/questions/40365741/default-values-for-vue-component-props-how-to-check-if-a-user-did-not-set-the